docs/source/optimizations/ibm/ibmq_compile_css.ipynb

```diff
@@ -22,21 +22,27 @@
    ]
```

Comment:
<p>This should be the second circuit: </p><pre class="ql-syntax" spellcheck="false">Line #1.    # Visualize the pulse sequence for the second compiled circuit
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Fixed.